js/okex.js

```diff
@@ -3355,6 +3370,50 @@ module.exports = class okex extends Exchange {
         };
     }
 
+    async fetchTransfers (code = undefined, since = undefined, limit = undefined, params = {}) {
+        if (!('transId' in params)) {
+            throw new ArgumentsRequired (this.id + ' fetchTransfers requires a transfer id param');
+        }
```

Comment:
If they require a transfer id, then this method is for a single transfer (`fetchTransfer`) not for a history of multiple transfers (`fetchTransfers`). If that is the case, I suggest to rename it to `fetchTransfer` and add the first mandatory argument `id` to it. Also, it is not entirely clear, why a currency parameter would be an optional parameter if the `transId` is specified. Usually, when it comes to a single historical entry, like one transfer or one order, they either always require a currency or don't accept it at all, cause the entry id points to the exact entry. We will need to resolve this.

Reply:
Okay I made these changes